front/components/assistant/conversation/ConversationContainer.tsx

```diff
@@ -195,22 +195,14 @@ export function ConversationContainer({
     )
   );
 
+  const addMentionRef = useRef<(mention: AgentMention) => void>(() => {});
```

Comment:
You can use `setSelectedAssistant` rather than passing a ref several levels down.